_config.yml

```diff
@@ -1,17 +1,17 @@
 name: The Rust Programming Language
-stable: "1.31.1"
-stable_date: "2018-12-20"
-stable_blog: "https://blog.rust-lang.org/2018/12/20/Rust-1.31.1.html"
-stable_full_version: "rustc 1.31.1 (b6c32da9b 2018-12-18)"
-beta: "1.32"
+stable: "1.32.0"
+stable_date: "2019-01-17"
+stable_blog: "https://blog.rust-lang.org/2019/01/17/Rust-1.32.0.html"
+stable_full_version: "rustc 1.32.0 (9fda7c223 2019-01-16)"
+beta: "1.33"
 beta_date: "2019-01-18"
 nightly: "1.33"
 channels:
   - name: "stable"
-    vers: "1.31.0"
-    package: "1.31.0"
+    vers: "1.32.0"
+    package: "1.32.0"
   - name: "beta"
-    vers: "1.32"
+    vers: "1.33"
```

Comment:
Actually, we should bump nightly to 1.34 while we're at it.

Reply:
`rustup` says 1.33 tho.. we can always make a new PR once we bump to 1.34?